ev3dev2/motor.py

```diff
@@ -76,16 +76,46 @@
     raise Exception("Unsupported platform '%s'" % platform)
 
 
+class LineFollowError(Exception):
+    """
+    Raised when a line following robot has lost the line
+    """
+    pass
+
+
+class LineFollowErrorTooFast(Exception):
```

Comment:
We probably need an example in the method's docstring which uses it and shows how to handle these.